Sources/Web3Core/Oracle/GasOracle.swift

```diff
@@ -99,24 +99,24 @@ final public class Oracle {
     /// - Returns: `[percentile_1, percentile_2, percentile_3, ...].count == self.percentile.count`
     /// by default there's 3 percentile.
     private func suggestTipValue() async throws -> [BigUInt] {
-        var rearrengedArray: [[BigUInt]] = []
+        var rearrangedArray: [[BigUInt]] = []
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.